react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class NewsLetterForm extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {value: ''};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event){
+    this.setState({value: event.target.value});
+  }
+
+  handleSubmit(event){
+    console.log("State ", {Email:this.state.value});
```

Comment:
A bit overkill on the `console.log` you could of just done `this.state.value` without the object